### PR TITLE
Add "interface wireless" to API data

### PR DESCRIPTION
**Pull request by hansmi**
##### SUMMARY
Implement support for the `interface wireless` configuration path by recording available fields. The default values match those of RouterOS 7.11 on a Mikrotik RB924i-2nD-BT5&BG77 ("Knot"). They're cross-checked with [upstream documentation](https://help.mikrotik.com/docs/display/ROS/Wireless+Interface).

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
community.routeros

### CODE REVIEW

**Comment by felixfontein** [comment]
Thanks for your contribution! Can you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks.

CC @derdeagle

**Comment by hansmi** [comment]
> Can you please add a changelog fragment?

Done. Please take another look.

File: changelogs/fragments/195-add-interface-wireless-data.yml
Comment (felixfontein) [inline changelogs/fragments/195-add-interface-wireless-data.yml:2]:
```suggestion
  - api_info, api_modify - add support for the ``interface wireless`` path (https://github.com/ansible-collections/community.routeros/pull/195).
```
`api_find_and_modify` does not use plugins/module_utils/_api_data.py.

**Comment by felixfontein** [comment]
@hansmi BTW, do you know why GitHub shows you as a "Contributor" for this repository, while at the same time saying that this is your first PR in this repository, and https://github.com/ansible-collections/community.routeros/commits?author=hansmi is empty? Did you change your GH user name at some point (and parts of GH remember, while others do not)?

**Comment by hansmi** [comment]
@felixfontein, I never changed my username and can't explain why it'd consider me a contributor. I did [contribute to the main `ansible/ansible`](https://github.com/ansible/ansible/commits?author=hansmi) repository a few years ago, but the commit history for the collection doesn't contain those.

File: changelogs/fragments/195-add-interface-wireless-data.yml
Comment (hansmi) [reply changelogs/fragments/195-add-interface-wireless-data.yml]: Ah, right. It only imports `split_path`. I misread the code. Fixed.

**Comment by felixfontein** [comment]
@hansmi hmm, I guess it's a bug in GitHub then :) The "funny" part is that due to GH recognizing you as a contributor, it automatically runs the GHA jobs even though this is your first PR. Usually it only runs them automatically without manual approval if someone already has a merged PR (or authored a commit, no idea what it exactly checks). But :shrug: at least in this case I have to press less buttons ;-)

**Comment by hansmi** [comment]
@felixfontein, it might be a bug indeed. I saw the "needs approval" message at first and thought you'd approved when I saw the actions running later. It might have been after I force-pushed a small modification.

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
Thanks!

**Pull request merged**
